src/huggingface_hub/__init__.py

```diff
@@ -142,7 +142,7 @@
         "RepoUrl",
         "UserLikes",
         "add_space_secret",
-        "change_discussion_status",
```

Comment:
Typo on this line. You might have forgotten a comma `,` between two lines and the linting merged the two strings in one.

Reply:
Oh, yes. Apologies!

The same error was also present a little bit further down. Corrected as well.